core/src/test/java/org/infinispan/test/AbstractInfinispanTest.java

```diff
@@ -91,16 +91,16 @@ protected void eventuallyEquals(int expected, IntSupplier supplier) {
             () -> expected == supplier.getAsInt());
    }
 
-   protected void eventuallyEquals(String message, long expected, LongSupplier supplier) {
+   public static void eventuallyEquals(String message, long expected, LongSupplier supplier) {
```

Comment:
The second option doesn't work since one test extends ```SingleCacheManagerTest``` and the other ```MultipleCacheManagersTest```

changing to a separate class looks better to me, but I prefer to do that in another JIRA than full this one with changes everywhere :)